arch/arm64/boot/dts/qcom/msm8953.dtsi

```diff
@@ -633,6 +633,24 @@
 		reg = <0 0 0 0>;
 	};
 
+	mpm: interrupt-controller@601d4 {
+		compatible = "qcom,mpm";
+		qcom,rpm-msg-ram = <&apss_mpm>;
+		interrupts = <GIC_SPI 171 IRQ_TYPE_EDGE_RISING>;
+		mboxes = <&apcs 1>;
+		interrupt-controller;
+		#interrupt-cells = <2>;
```

Comment:
```suggestion
		#interrupt-cells = <3>;
```
https://git.codelinaro.org/clo/la/kernel/msm-4.9/-/blob/LA.UM.9.6.2.r1-04800-89xx.0/arch/arm64/boot/dts/qcom/msm8953.dtsi#L268

Reply:
But i am not totally sure because of this https://git.codelinaro.org/clo/la/kernel/msm-4.9/-/blob/LA.UM.9.6.2.r1-04800-89xx.0/arch/arm64/boot/dts/qcom/msm8953.dtsi#L275

Reply:
I think the downstream mpm controller "#interrupt-cells = <3>" only because it is used as interrupt-parent of the whole device-tree. In the examples in the mainline devicetree "qcom,mpm" is use with "#interrupt-cells = <2>" (in sm6375.dtsi, qcm2290.dtsi).